lib/src/main/java/io/apptik/widget/MultiSlider.java

```diff
@@ -1035,8 +1037,10 @@ public boolean onTouchEvent(MotionEvent event) {
                         invalidate(currThumb.getThumb().getBounds()); // This may be within the padding region
                     }
 
-                    setThumbValue(currThumb, getValue(event, currThumb), true);
+                    int value = getValue(event, currThumb);
+                    setThumbValue(currThumb, value, true);
                     attemptClaimDrag();
+                    mOnThumbValueChangeListener.onStartTrackingTouch(this, currThumb, value);
```

Comment:
we have a method onStartTrackingTouch where thumbs are set as being dragged and their state is changed.
This is the place where the callback should be called. 
in fact 'attemptClaimDrag()' should also be called there